cli/docker/app/factory.go

```diff
@@ -8,10 +8,12 @@ import (
 	"github.com/docker/libcompose/project"
 )
 
-type ProjectFactory struct {
+// DockerProjectFactory is a struct that hold the app.ProjectFactory implementation.
+type DockerProjectFactory struct {
```

Comment:
I renamed that as it felt a little weird to have the exact same name for the interface and the struct *implementing* it. If it breaks outside uses of the lib, tell me, I'll revert that change.